test/fit_tests/config/stack11.json

```diff
@@ -0,0 +1,23 @@
+{
+  "11": {
```

Comment:
This is a override file. Any keys in an override file replace entries in the stack_config file. This is described in the config/README.md.